jsonschema-core.md

```diff
@@ -523,11 +523,23 @@ to inform the user of the evaluation path that produced the error or annotation.
 
 ### Keyword Interactions
 
-Keyword behavior MAY be defined in terms of the annotation results of
-[subschemas](#root) and/or adjacent keywords (keywords within the same schema
-object) and their subschemas. Such keywords MUST NOT result in a circular
-dependency. Keywords MAY modify their behavior based on the presence or absence
-of another keyword in the same [schema object](#schema-document).
+Keywords MAY modify their behavior based on the presence or absence of another
+keyword in the same [schema object](#schema-document). Such keywords MUST NOT
+result in a circular dependency.
+
+Unless otherwise specified, keywords act independently.
+
+#### Annotations as an Interaction Mechanism
+
+This document uses annotations as a mechanism to describe the interactions
+between keywords, however this is for illustrative purposes and not
+prescriptive. Implementations SHOULD use whatever mechanisms makes sense given
```

Comment:
I'm not sure that using "SHOULD" makes sense here because it's telling you a specific thing that you should be doing. Maybe this makes more sense as "should". I think "MAY" would also work.

Reply:
I'll go with MAY.